src/javascript/namespace-scanner.ts

```diff
@@ -29,16 +30,133 @@ export class NamespaceScanner implements JavaScriptScanner {
       document: JavaScriptDocument,
       visit: (visitor: Visitor) => Promise<void>) {
     const visitor = new NamespaceVisitor(document);
```

Comment:
Now that there are two visitors, let's call this one `namespaceVisitor` or `namespaceFinder`